Snowcast/RequestSnowData.swift

```diff
@@ -1,28 +1,32 @@
-import Foundation
+import PromiseKit
+import CoreLocation
 import Alamofire
 
-let SNOCOUNTRY_API_KEY: String = "SnoCountry.example"
-let SNOCOUNTRY_BASE_URL: String = "http://feeds.snocountry.net/conditions.php"
-let SNOWCOUNTRY_REGION_TYPE: String = "northeast"
-
-typealias snowConditionCompletion = (Dictionary <String, AnyObject>) -> ()
-typealias weatherCompletion = (Dictionary <String, AnyObject>) -> ()
+let BASE_URL: String = "http://localhost:9999/api/near"
```

Comment:
Name Format Violation: Variable name should only contain alphanumeric characters: 'BASE_URL'